### PR TITLE
Add the ability to turn on debug output

### PR DESCRIPTION
**Pull request by sethvargo**
Requires https://github.com/hashicorp/atlas-go/pull/20

/cc @mitchellh

### CODE REVIEW

**Comment by sethvargo** [comment]
@ryanuber do you want to look at this one real fast too please :smile:

File: cli.go [hunk 1]
```diff
@@ -137,15 +167,16 @@ Usage: %s [options] app path
...
+  -exclude=<path>     Glob pattern of files or directories to exlude (this may
+                      be specified multiple times)
+  -include=<path>     Glob pattern of files/directories to include (this may be
+                      specified multiple times, any excludes will override
+                      conflicting includes)
   -address=<url>      The address of the Atlas server
   -token=<token>      The Atlas API token
   -vcs                Use VCS to determine which files to include/exclude
 
+  -debug              Turn on debug putput
```
Comment (ryanuber) [inline cli.go]: Should be "output"

Comment (sethvargo) [inline cli.go]: You don't like putput? I thought it was cute.

Lol jk

**Comment by ryanuber** [comment]
Just a spelling mistake. Also, I'm assuming the two other options coming into the help were already implemented and the help just forgotten.

**Comment by sethvargo** [comment]
@ryanuber they were already there (look above). My OCD wouldn't permit some flags to end in periods and some to not, so I went ahead and fixed that.

**Comment by ryanuber** [comment]
Oh haha, yep LGTM!

**Pull request merged**